doc/Security.xml

```diff
@@ -4875,20 +4875,15 @@
     "roles":["onvif:Operator"]
 }
 ]]></programlisting>
-    <para>which is base64-encoded as</para>
+    <para>which is base64url-encoded in three parts according to RFC 7519 as</para>
     <programlisting>eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJpc3MiOiJzZXJ2ZXIucGF0aC5jb20iLCJuYmYiOjE2NzA0ODI4MDAsImV
-4cCI6MTY3MDU3MjgwMCwiYXVkIjoidGFyZ2V0LmF1ZGllbmNlIiwicm9sZXMiOlsib252aWY6T3BlcmF0b3IiXX0=.SflKxw
+4cCI6MTY3MDU3MjgwMCwiYXVkIjoidGFyZ2V0LmF1ZGllbmNlIiwicm9sZXMiOlsib252aWY6T3BlcmF0b3IiXX0.SflKxw
 RJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c</programlisting>
-    <para>While the individual components of the JWT are already base64-encoded, the whole structure is not technically base64-encoded. Therefore it is necessary to base64-encode it again, leading to</para>
-    <programlisting>ZXlKMGVYQWlPaUpLVjFRaUxDSmhiR2NpT2lKRlV6STFOaUo5LmV5SnBjM01pT2lKelpYSjJaWEl1Y0dGMGFDNW
-piMjBpTENKdVltWWlPakUyTnpBME9ESTRNREFzSW1WNGNDSTZNVFkzTURVM01qZ3dNQ3dpWVhWa0lqb2lkR0Z5
-WjJWMExtRjFaR2xsYm1ObElpd2ljbTlzWlhNaU9pSnZiblpwWmpwUGNHVnlZWFJ2Y2lKOS5TZmxLeHdSSlNNZU
-tLRjJRVDRmd3BNZUpmMzZQT2s2eUpWX2FkUXNzdzVj</programlisting>
   </appendix>
   <appendix xml:id="_Ref460769599">
     <title>JWT over SCTP example</title>
     <para>JWT, whose content example is shown in <xref xmlns:xlink="http://www.w3.org/1999/xlink"
-        linkend="q5l_q5m_51c"/>, can be used in the WS-Security header of the SOAP request:</para>
+        linkend="q5l_q5m_51c"/>, can be used in the WS-Security header of the SOAP request. To conform to the BinarySecurityToken format, the JWT is base64-encoded a second time.</para>
```

Comment:
@ocampana-videotec do you know if that part is actually true? I've assumed the double-base64 part was added due to this section, but I'm not actually sure. Do you remember if that was the reason? Or could we use the standard JWT shape here too?

Reply:
@jmelancongen  I confirm that the reason for the second encoding is the dot.